src/Input/Select/Select.test.tsx

```diff
@@ -480,6 +480,78 @@ describe('<Select/> prop onInputChange', () => {
   });
 });
 
+describe('<Select />', () => {
```

Comment:
maybe you can put all the new tests in this describe block and describe it as `<Select /> click to close option dropdown`?

Reply:
I'd like to just rename this test block as you suggested but without moving the other new tests to it. Because the others are in `<Select disableTypeing />` and `<Select defaultOpen />` which I think is specified enough.